website_sale_menu_partner_top_selling/models/res_config_settings.py

```diff
@@ -0,0 +1,30 @@
+# Copyright 2024 Tecnativa - Pilar Vargas
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class ResConfigSettings(models.TransientModel):
+
+    _inherit = "res.config.settings"
+
+    regular_products_limit = fields.Integer(default=10)
+
+    @api.model
+    def get_values(self):
+        res = super().get_values()
+        res.update(
+            regular_products_limit=int(
+                self.env["ir.config_parameter"]
+                .sudo()
+                .get_param("website_sale_menu_partner_top_selling.limit", 10)
+            ),
+        )
+        return res
+
+    def set_values(self):
+        res = super().set_values()
+        self.env["ir.config_parameter"].sudo().set_param(
+            "website_sale_menu_partner_top_selling.limit", self.regular_products_limit
+        )
+        return res
```

Comment:
I think this can be simplified as follows or defined as a field in the website model instead of a parameter. What do you think?
https://github.com/odoo/odoo/blob/15.0/odoo/addons/base/models/res_config.py#L339C48-L339C78

```suggestion
    regular_products_limit = fields.Integer(default=10, config_parameter='website_sale_menu_partner_top_selling.limit')

```

Reply:
thanks!